python/aswfdocker/tests/test_utils.py

```diff
@@ -140,7 +140,8 @@ def test_cli_download(self):
                 catch_exceptions=False,
             )
             self.assertEqual(result.exit_code, 0)
-            self.assertEqual(result.output, f"{tmpdirname}/packages/2019/tbb.tar.gz")
+            path = os.path.join(tmpdirname, "packages", "2019", "tbb.tar.gz")
```

Comment:
Could also do the following but it feels messy:
```suggestion
            path = os.path.join(tmpdirname, *"packages/2019/tbb.tar.gz".split("/"))
```